src/Umbraco.Core/Persistence/Migrations/Initial/BaseDataCreation.cs

```diff
@@ -268,7 +268,7 @@ private void CreateCmsDataTypePreValuesData()
 
         private void CreateUmbracoRelationTypeData()
         {
-            _database.Insert("umbracoRelationType", "id", false, new RelationTypeDto { Id = 1, Alias = Constants.Conventions.RelationTypes.RelateDocumentOnCopyAlias, ChildObjectType = new Guid(Constants.ObjectTypes.Document), ParentObjectType = new Guid("C66BA18E-EAF3-4CFF-8A22-41B16D66A972"), Dual = true, Name = Constants.Conventions.RelationTypes.RelateDocumentOnCopyName });
+            _database.Insert("umbracoRelationType", "id", false, new RelationTypeDto { Id = 1, Alias = Constants.Conventions.Relations.RelateDocumentOnCopy, ChildObjectType = new Guid(Constants.ObjectTypes.Document), ParentObjectType = new Guid(Constants.ObjectTypes.Document), Dual = true, Name = "Relate Document On Copy" });

```

Comment:
Once you add a new default RelationType you will also need to bump the IndentitySeed for the table.
That is done in this class where IdentitySeed is currently set to 2 in the attribute for id: https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/src/Umbraco.Core/Models/Rdbms/RelationTypeDto.cs